framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2884,17 +2886,34 @@ protected Set<? extends AnnotationMirror> getExceptionParameterLowerBoundAnnotat
      * #getThrowUpperBoundAnnotations()}.
      *
      * @param tree ThrowTree to check
+     * @param mtree MethodTree to look up annotations
      */
-    protected void checkThrownExpression(ThrowTree tree) {
+    protected void checkThrownExpression(ThrowTree tree, @Nullable MethodTree mtree) {
         AnnotatedTypeMirror throwType = atypeFactory.getAnnotatedType(tree.getExpression());
         Set<? extends AnnotationMirror> required = getThrowUpperBoundAnnotations();
+        if (mtree != null && getExceptionList(mtree) != null) {
+            List<AnnotatedTypeMirror> exceptionList = getExceptionList(mtree);
+            //            boolean exceptionFound = false;
```

Comment:
What are these commented-out blocks for, here and below?

Reply:
Already removed them.